### ISSUE TITLE
cxx export error

### ISSUE THREAD

**Issue by Zibri** [xyproto/cxx#3]
$ cxx export
make: *** No rule to make target 'export'.  Stop.

**Comment by xyproto** [comment]
Thanks for reporting! There were issues with the `export` command, so I temporarily removed it.

I'll add it back.

Would it be best with the old behavior, or with a new behavior where a `CMakeLists.txt` file were generated instead?

**Comment by Zibri** [comment]
I really don't know, but I would make it generate a normal Makefile and optionally files for cmake or autogen... I never used in the past... but I am getting lazy and I found this little jewel.

Another minor problem is this:
I used cxx to compile a small program I made which outputs random numbers as fast as possible.
in this case the "cxx opt" fails because my program never ends and also while testing it outputs everything on the screen...
I think the same would happen compiling "cat" or "dd" with cxx...
Regards and keep it up!
If you are interested: https://github.com/Zibri/rand
